tools/vulnerability-scan/wss-scan.sh

```diff
@@ -21,6 +21,13 @@
 
 set -e
 
+# Generate temporary `settings.gradle` file based on the name in `build.gradle`
+function generate_settings_gradle() {
+  settings_gradle_content=`cat build.gradle | grep name | grep -i $repo | head -n 1 | sed 's/name/rootProject.name =/g' | awk '{$1=$1};1'`
+  echo $settings_gradle_content > settings.gradle
+  cat settings.gradle
+}
+
 java -version
```

Comment:
Sorry, one more.
Please move this to the same line with `gradle -v ......`.
Thanks.

Reply:
But I think checking java is different from checking other installation. 
Java is needed to run `wss-unified-agent.jar` so that's why we had a check here before download it. Moving to one line and checking them all together seem to be meaningless from my perspective. 

Or we could actually remove `gradle -v ...` because I had that line just to double check everything is properly installed and present.

Reply:
I didnt realize you are having a if condition afterwards, you can just remove everything afterwards since you have `set -e` set.

If java not exist it will fail instantly, therefore your if statement is useless anyway.

Reply:
If you really really really want a statement just write:
```
! (java -version1> /dev/null 2>&1) || (echo Java not found && exit 1)
```

Reply:
Make sense to me. That check has been removed.